cluster/olm/ceph/generate-rook-csv.sh

```diff
@@ -68,67 +67,75 @@ if [[ -z $3 ]]; then
 fi
 ROOK_OP_VERSION=$3
 
-DEFAULT_CSV_FILE_NAME="${CSV_PATH}/ceph/${VERSION}/ceph.v${VERSION}.clusterserviceversion.yaml"
-DESIRED_CSV_FILE_NAME="${CSV_PATH}/rook-ceph.v${VERSION}.clusterserviceversion.yaml"
-if [[ -f "$DESIRED_CSV_FILE_NAME" ]]; then
-    echo "$DESIRED_CSV_FILE_NAME already exists, not doing anything."
-    exit 0
-fi
-
 #############
 # VARIABLES #
 #############
 YQ_CMD_DELETE=($yq delete -i)
-YQ_CMD_MERGE_OVERWRITE=($yq merge --inplace --overwrite --append)
-YQ_CMD_MERGE=($yq merge --inplace --append)
-YQ_CMD_WRITE=($yq write --inplace)
-OP_SDK_CMD=($operator_sdk olm-catalog gen-csv --csv-version)
+YQ_CMD_MERGE_OVERWRITE=($yq merge --inplace --overwrite --append -P )
```

Comment:
What does `-P` do? Does it have a full detailed version like `--inplace`?

Reply:
yes it is "--prettyPrint"

Reply:
I put it there, because it is easier to look at the differences with a full yaml syntax than the json one-line syntax.

Reply:
Let's use `--prettyPrint` then. Thanks.

Also is the white space needed after it?

Reply:
Done